cmd/rook/ceph/sanitize_disk.go

```diff
@@ -0,0 +1,219 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ceph
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"sync"
+
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	"github.com/rook/rook/pkg/clusterd"
+	"github.com/rook/rook/pkg/daemon/ceph/osd"
+	oposd "github.com/rook/rook/pkg/operator/ceph/cluster/osd"
+)
+
+const (
+	ddBS    = "1M"           // DD's block size
+	ddCount = "10"           // DD runs over the first 10 offsets
+	ddFlags = "direct,dsync" // DD's sync flags"
+)
+
+// DiskSanitizer is simple struct to old the context to execute the commands
+type DiskSanitizer struct {
+	context          *clusterd.Context
+	sanitizeDiskSpec *cephv1.SanitizeDiskSpec
+	disk             string
+}
+
+func newDiskSanitizer(context *clusterd.Context) DiskSanitizer {
+	return DiskSanitizer{
+		context: context,
+		sanitizeDiskSpec: &cephv1.SanitizeDiskSpec{
+			SanitizeMethod:     cephv1.SanitizeMethodProperty(sanitizeMethod),
+			SanitizeUtility:    cephv1.SanitizeUtilityProperty(sanitizeUtility),
+			SanitizeDataSource: cephv1.SanitizeDataSourceProperty(sanitizeDataSource),
+			SanitizeIteration:  &sanitizeIteration,
+		},
+	}
+}
+
+func startSanitizeDisk(context *clusterd.Context) {
```

Comment:
We're really sanitizing multiple disks, not just one
```suggestion
func startSanitizeDisks(context *clusterd.Context) {
```